Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -616,8 +660,12 @@ def main():
         print ("Cleaning completed successfully !\n")
         return 0
 
+    if args.btg_profile in ["vm","fvme"] and args.tpm == "none":
+        print "ERROR: Choose appropriate Tpm type for BootGuard profile 3 and 5"
+        return 0
+
     print ("Executing stitch.......")
-    if stitch (stitch_dir, args.stitch_zip, args.btg_profile, args.quad_mode, args.plat_data, args.platform):
+    if stitch (stitch_dir, args.stitch_zip, args.btg_profile, args.quad_mode, args.plat_data, args.platform, args.tpm):
```

Comment:
There is a example line in the beginning of the script. Can we update that too since we are adding this new argument for Tpm Type?

Reply:
will add.